### PR TITLE
Card now extends to show additional information. Remove search compon…

### PR DESCRIPTION
**Pull request by mike-duke**
…ent from landingScreen

Extending the card now works properly. Needs styling. Now that we have it working, we can move the pieces around in any way we like. Let me know what you think.

### CODE REVIEW

**Comment by mike-duke** [comment]
i also fixed a bug in the filtering for the distance and difficulty dropdowns

File: src/styles/_Card.scss [hunk 1]
```diff
@@ -31,11 +31,17 @@ $card-text-color: #FFFF00;
   color: white;
 }
 
+
```
Comment (pvinthemix) [inline src/styles/_Card.scss:34]: Really like this feature of extending a class in CSS

File: src/styles/_Card.scss [hunk 2]
```diff
@@ -31,11 +31,17 @@ $card-text-color: #FFFF00;
...
 .full-card {
   @extend .card;
   height: 400px;
 }
 
+.card:hover {
+  cursor: pointer;
+  animation: bounce .9s;
```
Comment (pvinthemix) [inline src/styles/_Card.scss:42]: I may make a change here to make the hover a border, so it looks like the road signs we were going for.

File: src/Controls.js [hunk 3]
```diff
@@ -32,17 +39,22 @@ export default class Controls extends Component {
...
+  }
+
   resetApplication = () => {
     this.props.toggleLandingScreen();
   }
 
   render() {
     return (
       <div className="controls-container">
+        <h4>Number of Trails: {this.props.trails.length}</h4>
```
Comment (pvinthemix) [inline src/Controls.js:53]: Looks like you were able to display the number of trails that populate after rendering. Glad we went this route.

File: src/App.js [hunk 4]
```diff
@@ -121,6 +130,7 @@ class App extends Component {
             <Controls fetchTrails={this.fetchTrails} 
                       filterByDistance={this.filterByDistance}
                       filterByDifficulty={this.filterByDifficulty}
+                      filterByParkName={this.filterByParkName}
```
Comment (pvinthemix) [inline src/App.js:133]: Glad we went with the drop down here. I think the functionality fits well.

**Pull request merged**